src/components/sbb-icon/readme.md

```diff
@@ -0,0 +1,69 @@
+The `sbb-icon` component provides a solid way of rendering registered and named icons. The component will dynamically load an SVG for each icon, avoiding multiple requests to the same icon. The icon components are not tied to specific icon sets; you can register a new SVG icon or a custom namespace and then provide the `sbb-icon` with the `name` property in the format `name="icon-name"` or `name="namespace:icon-name"`. Note that if you do not provide a namespace, the default namespace "sbb" will be used pointing to the SBB icon CDN. 
```

Comment:
Maybe add the SBB CDN at the end could be useful for consumers? so they can see if those icons fit their needs..

Reply:
Of course, it could be useful. Thank you.